draft-wing-cidfi.md

```diff
@@ -711,6 +711,9 @@ client                           edge router           server
 flow shows an initial QUIC handshake for simplicity (steps 1 and 7)
 but because of QUIC connection migration ({{Section 9 of QUIC}}) the
 QUIC messages might appear later.
+> Also, "Map DCID=xyz as high importance" refers to a CID chosen by
+> the server not the DCID used by the server to communicate with
+> the client.
```

Comment:
I don't get the new line. QUIC server designates the connection ID which the client uses to send packets to the server, for example see the packet capture in https://www.cloudshark.org/captures/3adfd15940d0

Reply:
This  is just about "direction" :-)

We do have both source and destination CIDs that depends on the direction of a packet. The SCID used by a client will be used as DCID for packets sent by a server and vice versa. So, if we want "map DCID", we need to clarify the direction.

Reply:
No, SCID will be used for the packets sent from the server and DCID will be used for the packets sent from the client and the CID is generated by the server.

Reply:
> No

How is this relevant to what I said? :-) 

What I'm saying is that "DCID" are per direction. Once that clarified, we can add another clarification whether the specified differentiated behavior will apply or not for both directions of a flow or only if explicitly requested by the client. If the policy is bidir, what you indicated will be followed by a CNE.

Till now, we don't have all these details. I know this is digging a little bit into much details, but I'm afraid that we need to have things clear.